lua/gruvbox_crisp/groups.lua

```
@@ -0,0 +1,271 @@
+    
+    -- Python-specific highlights
+    ["@decorator.python"]          = { fg = p.decorator },
+    ["@variable.language.self.python"] = { fg = p.const, italic = true },
+    ["@variable.language.cls.python"]  = { fg = p.const, italic = true },
+    ["@function.builtin.python"]   = { fg = p.preproc },
+    ["@function.magic.python"]     = { fg = p.type },
+    ["@string.documentation.python"] = { fg = p.comment, italic = bool(o.italics.comments) },
+    ["@type.annotation.python"]    = { fg = p.type },
+    ["@keyword.storage.python"]    = { fg = p.kw_ctrl },  -- class, def, async
+    ["@parameter.python"]          = { fg = p.property },
+    
```

Luacheck (1.2.0)

[warning] line contains only whitespace

(W611)